R/ipr2viz.R

```diff
@@ -13,17 +13,17 @@
 #################################
 ## Modified gggenes::theme_genes
 #################################
-## theme_genes2 adapted from theme_genes (w/o strip.text())
+## themeGenes2 adapted from theme_genes (w/o strip.text())
 ## https://github.com/wilkox/gggenes/blob/master/R/theme_genes.R
-#' Theme Genes2
+#' themeGenes2
```

Comment:
I made some slight adjustments -- for consistency we're going to begin titling .Rd functions the same as the function call.